src/js/units.js

```diff
@@ -0,0 +1,85 @@
+export const UNITS = {
+    length: {
+        angstrom: "ang",
+        bohr: "bohr",
+        picometer: "pm",
+        nanometer: "nm",
+        micrometer: "µm",
+        millimeter: "mm",
+        meter: "m",
+    },
+    energy: {
+        hartree: "hartree",
+        electronvolt: "eV",
+        kJ_mol: "kJ/mol",
+        kcal_mol: "kcal/mol",
+        wavenumber: "cm-1",
+        joule: "J",
+    },
+    mass: {
+        kilogram: "kg",
+        dalton: "Da",
+        atomicMassUnit: "amu",
+    },
+};
+
+/**
+ * Conversion coefficients based on data taken from Wikipedia (date accessed: 05/20/2022)
```

Comment:
This would need a better set of references ...

Reply:
I added the links to the corresponding Wikipedia pages for reference.